pack/valk_encounter.json

```
@@ -26,6 +28,7 @@
 		"scheme": 2,
 		"set_code": "valk_nemesis",
 		"set_position": 1,
+		"text": "<b>When Revealed</b>: Search the encounter deck, discard pile, and set-aside area for a copy of Seduced and attach it to your identity. (Shuffle.)",
```

It seems the uses of "_(Shuffle.)_" are defined in italic.

```suggestion
		"text": "<b>When Revealed</b>: Search the encounter deck, discard pile, and set-aside area for a copy of Seduced and attach it to your identity. <i>(Shuffle.)</i>",
```